README.md

```diff
@@ -57,7 +57,7 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
     - [Serverless](#serverless)
     - [Testing](#testing)
     - [Wrappers](#wrappers)
-  - [Services based on Docker (:heavy_dollar_sign:)](#services-based-on-docker-heavy_dollar_sign)
+  - [Services based on Docker (:heavy_dollar_sign:)](#services-based-on-docker-heavydollarsign)
```

Comment:
Thank you for your contribution, appreciate the addition of TrendMicro. It would be great if you could revert this change (navigation) as it breaks the link to the section, other than that everything looks fine. Thank you.

Reply:
Apologies! I have reverted the erroneous change.